src/seasidecache.cpp

```diff
@@ -170,6 +170,9 @@ QMap<QString, QString> managerParameters()
     QMap<QString, QString> rv;
     // Report presence changes independently from other contact changes
     rv.insert(QString::fromLatin1("mergePresenceChanges"), QString::fromLatin1("false"));
+    if (Q_UNLIKELY(!QString::fromUtf8(qgetenv("LIBCONTACTS_TEST_MODE")).isEmpty())) {
```

Comment:
There's no point in adding Q_UNLIKELY to some test that gets run once, it just makes it harder to read...